mender/templates/api-gateway-configmap.yaml

```diff
@@ -473,6 +495,15 @@ data:
         compression:
           compress: true
 
+        ratelimit:
+          ratelimit:
+            average: {{ .Values.api_gateway.ratelimit.average }}
+            burst: {{ .Values.api_gateway.ratelimit.burst }}
+            period: {{ .Values.api_gateway.ratelimit.period }}
+            sourcecriterion:
+              ipstrategy:
+                depth: 1
```

Comment:
Doesn't this parameter depend on the infrastructure you're running Mender on-prem? Should we make it configurable?